Makefile

```diff
@@ -54,13 +53,13 @@ pullmessages:
 
 setup:
 	virtualenv -p `which python3.8` $(VENV_DIR)
```

Comment:
I don't have virtualenv installed anymore, but will this still work if $(VENV_DIR) has been removed from line 4? Maybe we could switch to doing `python3.8 -m venv env` (I'll keep using my own pyenv workflow :) )

Reply:
Oh, just saw the comment at the top of the file! That would be fine by me too, and just remove this line?